experimental/Schemes/Resolution_structure.jl

```diff
@@ -1,54 +1,114 @@
-## Warnung: show auf desingMor geht noch nicht!!!
 export _desing_curve
 export  find_refinement_with_local_system_of_params
 
+##############################################################################
+## Concrete Type for normalization
+## very similar to CoveredSchemeMorphism, but allowing disjoint handling
+## of disjoint components
+##############################################################################
+@attributes mutable struct NormalizationMorphism{
+    DomainType<:AbsCoveredScheme,
+    CodomainType<:AbsCoveredScheme
+   } <:AbsCoveredSchemeMorphism{
+                                 DomainType,
+                                 CodomainType,
+                                 Nothing,
+                                 NormalizationMorphism,
```

Comment:
This looks a bit circular. What is going on?

Reply:
AbsCoveredSchemeMorphism needs 4 arguments: domain, codomain, base change (here nothing) and self.

Reply:
The requirements for implementing a `Hecke.Map` is what's going on here. One argument is the type itself.